include/reactor-cpp/trace.hh

```diff
@@ -66,56 +66,42 @@ TRACEPOINT_EVENT(
   )
 )
 
-TRACEPOINT_EVENT(
-  reactor_cpp,
-  schedule_action,
-  TP_ARGS(
-    const std::string&, reactor_name_arg,
-    const std::string&, action_name_arg,
-    const reactor::Tag&, tag_arg
-  ),
-  TP_FIELDS(
-    ctf_string(reactor_name, reactor_name_arg.c_str())
-    ctf_string(action_name, action_name_arg.c_str())
-    ctf_integer(unsigned long, timestamp_ns,
-                tag_arg.time_point().time_since_epoch().count())
-    ctf_integer(unsigned, timestamp_microstep, tag_arg.micro_step())
-  )
-)
+TRACEPOINT_EVENT(reactor_cpp, reaction_execution_finishes,
+                 TP_ARGS(int, worker_id_arg, const std::string&, reaction_name_arg),
+                 TP_FIELDS(ctf_string(reaction_name, reaction_name_arg.c_str())
+                               ctf_integer(int, worker_id, worker_id_arg)))
 
 TRACEPOINT_EVENT(
-  reactor_cpp,
-  trigger_reaction,
-  TP_ARGS(
-    const std::string&, reactor_name_arg,
-    const std::string&, reaction_name_arg,
-    const reactor::LogicalTime&, tag_arg
-  ),
-  TP_FIELDS(
-    ctf_string(reactor_name, reactor_name_arg.c_str())
-    ctf_string(reaction_name, reaction_name_arg.c_str())
-    ctf_integer(unsigned long, timestamp_ns,
-                tag_arg.time_point().time_since_epoch().count())
-    ctf_integer(unsigned, timestamp_microstep, tag_arg.micro_step())
-  )
-)
-
-#endif /* _REACTOR_CPP_TRACE_H */
+    reactor_cpp, schedule_action,
+    TP_ARGS(const std::string&, reactor_name_arg, const std::string&, action_name_arg, const reactor::Tag&, tag_arg),
+    TP_FIELDS(ctf_string(reactor_name, reactor_name_arg.c_str()) ctf_string(action_name, action_name_arg.c_str())
+                  ctf_integer(unsigned long, timestamp_ns, tag_arg.time_point().time_since_epoch().count())
+                      ctf_integer(unsigned, timestamp_microstep, tag_arg.micro_step())))
+
+TRACEPOINT_EVENT(reactor_cpp, trigger_reaction,
+                 TP_ARGS(const std::string&, reactor_name_arg, const std::string&, reaction_name_arg,
+                         const reactor::LogicalTime&, tag_arg),
+                 TP_FIELDS(ctf_string(reactor_name, reactor_name_arg.c_str()) ctf_string(reaction_name,
```

Comment:
Is there a way to disable auto formatting locally? Looks like clang-format does not play well with the TP_FIELDS macros.

Reply:
not sure we have line length 120 right ?